src/elm/Auth.elm

```diff
@@ -31,13 +31,24 @@ import Json.Encode as Encode exposing (Value)
 import List.Extra as LE
 import Log
 import Profile exposing (Profile)
-import Regex
 import Route
 import Session.Shared as Shared exposing (Shared)
 import Task
 import UpdateResult as UR
 import Utils
-import Validate exposing (Validator, fromValid, ifNotInt, validate)
+import Validate exposing (Validator, validate)
+
+
+{-| Authentication of a user with Passphrase or Private Key.
+
+  - Passphrase consists of 12 unique words given to the user during the registration. Only users knows this phrase.
+  - Private Key (PK) is a hashed analogue of a Passphrase, we identify the user by the PK in the backend.
```

Comment:
actually we identify the user by the Public Key, not the Private Key!

Reply:
Oh, I see 😅 Thank you.

So, here is the right flow: 12-words → private key → public key. And then we use the last one for communication with the backend. Right?

@lucca65 How do you think, was it a good idea to add this description? I think it may be useful for newcomers and for future us. As for me, I spent quite a time understanding the code and this kind of documentation could help me a lot.

Reply:
That's exactly right @amiskov! 💪 In the end the private key never go to the internet in any way or form. 

About the description: +1000 I love this approach of documenting as we go along, feels much more natural and helpful!